actions/admin/paymail_addresses.go

```diff
@@ -61,8 +79,20 @@ func (a *Action) paymailAddressesCount(w http.ResponseWriter, req *http.Request,
 }
 
 // paymailCreateAddress will create a new paymail address
+// Create Paymail godoc
+// @Summary      Create paymail
+// @Description  Create paymail
+// @Tags		 Admin
+// @Param key query string true "key"
```

Comment:
Probably there should be xpub instead of key.
"xpub := params.GetString("xpub")"